core/ipc/rpc/utils/src/main/java/org/opennms/core/rpc/utils/mate/EntityScopeProviderImpl.java

```diff
@@ -87,11 +89,73 @@ public Scope getScopeForNode(final Integer nodeId) {
                             .map("node", "sys-location", (n) -> Optional.ofNullable(n.getSysLocation()))
                             .map("node", "sys-contact", (n) -> Optional.ofNullable(n.getSysContact()))
                             .map("node", "sys-description", (n) -> Optional.ofNullable(n.getSysDescription()))
+                            .map("node", "sys-objectid", (n) -> Optional.ofNullable(n.getSysObjectId()))
                             .map("node", "location", (n) -> Optional.ofNullable(n.getLocation().getLocationName()))
                             .map("node", "area", (n) -> Optional.ofNullable(n.getLocation().getMonitoringArea()))
-            );
+                            .map("node", "category" , (n) -> Optional.ofNullable(node.getAssetRecord()).map(OnmsAssetRecord::getCategory))
```

Comment:
I would strongly advocate to put this into the "asset" context.

I would like to see the assets stored as meta-data itself. By putting it in its own context we can avoid a rename later on.

And we should update to docs for these additional keys.

Reply:
> I would strongly advocate to put this into the "asset" context.
Sure, I have no strong opinion on that. You propose to create a different scope "asset"?

> And we should update to docs for these additional keys.
:+1: will do that once we have settled on all attributes. There still might be a few missing.

Reply:
@fooker is that how you were envisioning the asset scope?